### ISSUE TITLE
new-app can't find imagestream for Jenkins example

### ISSUE THREAD

**Issue by stevekuznetsov** [openshift/origin#7023]
Deployment fails to trigger on image change, `oc status` claims no imagestream exists, but one does. This is the entire workflow:
```sh
$ wget https://github.com/openshift/origin/releases/download/v1.1.1.1/openshift-origin-server-v1.1.1.1-940be51-linux-64bit.tar.gz
$ tar -xzf openshift-origin-server-v1.1.1.1-940be51-linux-64bit.tar.gz
$ cd openshift-origin-server-v1.1.1.1-940be51-linux-64bit.tar.gz
$ sudo ./openshift start &
$ sudo chmod a+rw openshift.local.config/master/admin.kubeconfig
$ sudo chmod a+r openshift.local.config/master/openshift-registry.kubeconfig
$ oc login -u system:admin --config=openshift.local.config/master/admin.kubeconfig
$ oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
$ oc get service registry
apiVersion: v1
kind: Service
metadata:
  creationTimestamp: 2016-02-04T04:01:04Z
  labels:
    docker-registry: default
  name: docker-registry
  namespace: default
  resourceVersion: "306"
  selfLink: /api/v1/namespaces/default/services/docker-registry
  uid: e989c981-caf3-11e5-b2b1-005056b13f8f
spec:
  clusterIP: 172.30.233.15
  portalIP: 172.30.233.15
  ports:
  - name: 5000-tcp
    port: 5000
    protocol: TCP
    targetPort: 5000
  selector:
    docker-registry: default
  sessionAffinity: ClientIP
  type: ClusterIP
status:
  loadBalancer: {}
$ wget https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/jenkins-persistent-template.json
$ wget https://raw.githubusercontent.com/openshift/origin/master/examples/image-streams/image-streams-rhel7.json
$ oc new-project jenkins
$ oc policy add-role-to-user edit system:serviceaccount:test:default
$ oc describe serviceaccount default
Name:		default
Namespace:	jenkins
Labels:		<none>

Tokens:            	default-token-6tbo6
                   	default-token-psc4q

Image pull secrets:	default-dockercfg-tm6wr

Mountable secrets: 	default-token-psc4q
                   	default-dockercfg-tm6wr
$ oc create -f image-streams-rhel7.json
imagestream "ruby" created
imagestream "nodejs" created
imagestream "perl" created
imagestream "php" created
imagestream "python" created
imagestream "mysql" created
imagestream "postgresql" created
imagestream "mongodb" created
imagestream "jenkins" created
$ oc get is jenkins -w
NAME      DOCKER REPO                          TAGS      UPDATED
jenkins   172.30.233.15:5000/jenkins/jenkins             
jenkins   172.30.233.15:5000/jenkins/jenkins   1,latest   Less than a second ago
$ oc new-app jenkins-persistent-template.json 
--> Deploying template jenkins-persistent for "jenkins-persistent-template.json"
     With parameters:
      JENKINS_SERVICE_NAME=jenkins
      JENKINS_PASSWORD=password # generated
      VOLUME_CAPACITY=512Mi
--> Creating resources ...
    Service "jenkins" created
    Route "jenkins" created
    PersistentVolumeClaim "jenkins" created
    DeploymentConfig "jenkins" created
--> Success
    Run 'oc status' to view your app.
$ oc status
In project Jenkins Server Deployment (jenkins) on server https://152.3.53.106:8443

svc/jenkins - 172.30.241.21:8080
  dc/jenkins deploys openshift/jenkins:latest 
    #1 deployment waiting on image or update
  exposed by route/jenkins

Errors:
  * The image trigger for dc/jenkins will have no effect because is/jenkins does not exist.

View details with 'oc describe <resource>/<name>' or list everything with 'oc get all'.
$ oc get is/jenkins -o yaml
apiVersion: v1
kind: ImageStream
metadata:
  annotations:
    openshift.io/image.dockerRepositoryCheck: 2016-02-04T04:24:34Z
  creationTimestamp: 2016-02-04T04:24:15Z
  name: jenkins
  namespace: jenkins
  resourceVersion: "920"
  selfLink: /oapi/v1/namespaces/jenkins/imagestreams/jenkins
  uid: 26517cf6-caf7-11e5-bcd2-005056b13f8f
spec:
  tags:
  - from:
      kind: DockerImage
      name: registry.access.redhat.com/openshift3/jenkins-1-rhel7:latest
    name: "1"
  - from:
      kind: ImageStreamTag
      name: "1"
    name: latest
status:
  dockerImageRepository: 172.30.233.15:5000/jenkins/jenkins
  tags:
  - items:
    - created: 2016-02-04T04:24:34Z
      dockerImageReference: registry.access.redhat.com/openshift3/jenkins-1-rhel7:latest
      image: e9e1ffba0334722e8a5fa3a91b7fec78ac8133538399d65e5763a6b50288c6af
    tag: "1"
  - items:
    - created: 2016-02-04T04:24:34Z
      dockerImageReference: registry.access.redhat.com/openshift3/jenkins-1-rhel7:latest
      image: e9e1ffba0334722e8a5fa3a91b7fec78ac8133538399d65e5763a6b50288c6af
    tag: latest
```

/cc @bparees @gabemontero

**Comment by bparees** [comment]
It doesn't exist in the openshift namespace, which is where your DC(from
the Jenkins template) is looking for it.

create the image streams in the openshift namespace.

Ben Parees | OpenShift

**Comment by stevekuznetsov** [comment]
We need to make that more obvious from `oc status`. Thanks @bparees

**Comment by bparees** [comment]
@stevekuznetsov can you create a new issue for that?
